pyaerocom/web/helpers_evaluation_iface.py

```diff
@@ -467,6 +487,188 @@ def _init_ts_data():
         yearly_mod = []
         )
 
+def _create_diurnal_weekly_data_object(data,resolution):
+    import xarray as xr
+    if resolution == 'monthly':
+        b1 = 0
+        b2 = 0
+        step =1
+    elif resolution == 'bimonthly':
+        b1 = 0
+        b2 = 1
+        step = 2
+    elif resolution == 'seasonal':
+        b1 = 0
+        b2 = 2
+        step = 3
+        seasons = ['DJF','MAM','JJA','SON']
```

Comment:
With a single year of data, this will make the DJF season out of Jan, Feb and Dec of that year. Seasons are always tricky to work with for this exact reason, and I cleared this behaviour it with Hilde for our current use case.

Given more than a single year of data it will group all DJF months together and average them.